devices/src/virtio/mmio.rs

```diff
@@ -449,6 +489,19 @@ mod tests {
             self.acked_features = page + value;
         }
 
+        fn get_available_features(&self) -> u64 {
+            0
+        }
+
+        #[allow(clippy::cast_lossless)]
+        fn get_acked_features(&self) -> u64 {
+            self.acked_features as u64
```

Comment:
Why not `u64::from(self.acked_features)` in order to avoid the `#[allow(clippy::cast_lossless)]`?

Reply:
Yep, it's better with your suggestion. Updated.